gcp_variant_transforms/pipeline_common.py

```diff
@@ -199,6 +200,45 @@ def read_headers(pipeline, pipeline_mode, all_patterns):
   return headers
 
 
+def read_variants(pipeline,  # type: beam.Pipeline
+                  all_patterns,  # type: List[str]
+                  pipeline_mode,  # type: int
+                  allow_malformed_records,  # type: bool
+                  representative_header_lines=None,  # type: List[str]
+                  vcf_parser=vcfio.VcfParserType.PYVCF  # type: int
+                 ):
+  # type: (...) -> pvalue.PCollection
+  """Returns a PCollection of Variants by reading VCFs."""
+  compression_type = get_compression_type(all_patterns)
+  if compression_type == filesystem.CompressionTypes.GZIP:
+    splittable_bgzf = get_splittable_bgzf(all_patterns)
```

Comment:
Is get_splittable_bgzf being used outside pipeline common anymore? if not, add an underscore.

Reply:
Done. Thanks!